### ISSUE TITLE
Collapse arrow is flipped in RTL mode

### ISSUE THREAD

**Issue by gyoshev** [telerik/kendo-angular#1174]
## I'm submitting a...

* Bug report 

## Current behavior

In RTL, the Splitter panes are flipped, however the collapse arrows are not, and point in the wrong direction.

## Expected behavior

The arrows should point in the direction of a collapsible pane, and away from a collapsed pane. See the [Kendo UI for jQuery Splitter](https://demos.telerik.com/kendo-ui/splitter/right-to-left-support) for reference.

## Minimal reproduction of the problem with instructions

https://plnkr.co/edit/f8yBesXuNOvVdc08VvR4?p=preview

## What is the motivation or use case for changing the behavior?

Predictable Splitter behaviour in RTL scenarios

## Environment

Package versions:

```
@progress/kendo-angular-layout@1.2.0
@progress/kendo-theme-default@2.45.4
```

**Comment by gyoshev** [comment]
Workaround (and probably the fix):

```css
.k-splitter-flex.k-splitter-horizontal[dir="rtl"] > .k-splitbar > .k-collapse-prev,
.k-splitter-flex.k-splitter-horizontal[dir="rtl"] > .k-splitbar > .k-collapse-next {
    transform: scaleX(-1);
}
```

**Comment by elena-gancheva** [comment]
This issue is fixed and available in the latest dev version of the package - `v2.47.1-dev.201801181418`.

**Comment by gyoshev** [comment]
Officially released in `2.47.1`.
